capstone/src/main/java/com/google/sps/servlets/EntityRecognitionServlet.java

```diff
@@ -23,6 +22,7 @@
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
     String messages = request.getParameter("messages");
+    System.out.println("\n\n" + messages + "\n\n");
```

Comment:
Can we remove these printlns before submitting?

Reply:
we no longer need this servlet to call nlp api so will delete servlet